### PR TITLE
Created new playground cards

### PR DESCRIPTION
**Pull request by 06hpathak**

### CODE REVIEW

File: src/pages/components-overview/CardComponent.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Card, CardContent, Typography, Box } from '@mui/material';
+
+const CardComponent = ({ card }) => {
+  return (
+    <a href={card.link} target="_blank" rel="noopener noreferrer" style={{ textDecoration: 'none' }}>
```
Comment (thesiriusacademy) [inline src/pages/components-overview/CardComponent.js:6]: You can remove this `<a>` tag...just keep it simple Card

File: src/pages/components-overview/CardComponent.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+              </Typography>
+            </Box>
+            <Box>
+              <Typography variant="body2" component="div" sx={{ marginBottom: 1, fontSize: 14, textAlign: 'center' }}>
+                Start Date: {card.startDate}
+              </Typography>
+              <Typography variant="body2" component="div" sx={{ fontSize: 14, textAlign: 'center' }}>
+                End Date: {card.endDate}
+              </Typography>
+            </Box>
```
Comment (thesiriusacademy) [inline src/pages/components-overview/CardComponent.js:36]: Remove this...we dont need this

File: package.json [hunk 3]
```diff
@@ -9,6 +9,7 @@
     "@emotion/cache": "^11.11.0",
     "@emotion/react": "^11.11.3",
     "@emotion/styled": "^11.11.0",
+    "@mui/icons-material": "^5.15.15",
```
Comment (thesiriusacademy) [inline package.json:12]: We already have `@ant-design/icons`, why did we add this?

File: src/pages/components-overview/Playground.js [hunk 4]
```diff
@@ -1,252 +1,112 @@
...
+      endDate: '2022-09-20',
+      link: 'https://www.thesiriusacademy.com/pet-care'
+    },
+    {
+      title: 'Cafeteria Rush',
+      color: 'linear-gradient(to right, #ff5722, #009688)',
+      icon: <CafeIcon fontSize="large" sx={{ opacity: 0.5, fontSize: 100 }} />,
+      startDate: '2022-04-18',
+      endDate: '2022-11-01',
+      link: 'https://www.thesiriusacademy.com/cafeteria-rush'
```
Comment (thesiriusacademy) [inline src/pages/components-overview/Playground.js:65]: We do not need link over here, you can remove link.
Give each element an `id`

File: src/pages/components-overview/Playground.js [hunk 5]
```diff
@@ -1,252 +1,112 @@
...
-                <Typography variant="h6">Size: 14px</Typography>
-                <Typography variant="h6">Weight: Italic Regular & Italic Bold</Typography>
-                <Typography variant="h6">Line Height: 22px</Typography>
-              </Breadcrumbs>
-            </>
-          </MainCard>
-        </Stack>
-      </Grid>
+      {cardData.map((card, index) => (
+        <Grid item xs={12} sm={6} md={4} lg={3} key={index}>
```
Comment (thesiriusacademy) [inline src/pages/components-overview/Playground.js:104]: we should avoid giving `index` as key.
you can assign `id` to each card and use that id `key={card.id}`

**Review by thesiriusacademy — CHANGES_REQUESTED** [review]
added comments

File: package.json (hunk 3 above)
Comment (06hpathak) [reply package.json:12]: In cards we are using icons of mui

**Review by thesiriusacademy — APPROVED** [review]

**Pull request merged**